.buildkite/pipeline.trigger.compliance.tests.sh

```diff
@@ -26,8 +26,8 @@ steps:
 EOF
 
 # Generate each test we want to do.
-compliance_test 8.14.0-SNAPSHOT 3.1.5
-compliance_test 8.13.4 3.1.5
+compliance_test 8.14.1-SNAPSHOT 3.2.0
+compliance_test 8.14.0 3.2.0
```

Comment:
Should we use other versions or add a new compliance test to keep 3.1.5 ?

Reply:
Currently , package `log_synthetic_mode` fails with these versions with the following error:

```
Error: can't install the package: could not zip-install package; API status code = 500; response body = {"statusCode":500,"error":"Internal Server Error","message":"illegal_argument_exception\n\tRoot causes:\n\t\tillegal_argument_exception: field [decision_id] of type [text] doesn't support synthetic source unless it is stored or has a sub-field of type [keyword] with doc values or stored and without a normalizer"}
```